lib/vaahextendflutter/services/logging_library/logging_library.dart

```diff
@@ -77,7 +77,7 @@ class Log {
     bool disableCloudLogging = false,
   }) {
     if (_config.enableLocalLogs && !disableLocalLogging) {
-      Console.danger('$throwable\n$hint', data);
+      Console.danger('$throwable\n$hint\n$stackTrace', data);
```

Comment:
we should have StackTrace? stackTrace as argument and pass it separately, instead of passing it along with message.

I also suggest renaming, log function parameters to String? message, dynamic data/error, Stacktrace? stacktrace. If not understandable, can take this offline.

Reply:
Let me try, will update on this.